SiteLogToGeodesyML/sitelogtogeodesyml.py

```diff
@@ -1151,7 +1154,8 @@ def Begin(cls):
     NotesAddition = re.compile(r'^(\s{31,})(?P<value>.*)$', re.IGNORECASE)
 
     def __init__(self):
-        self.siteLocation = geo.siteLocationType()
+        text = "site-location"
+        self.siteLocation = geo.SiteLocationType(id=text)
```

Comment:
Any reason `self.siteLocation = geo.SiteLocationType(id="site-location")` wouldn't work?

Reply:
I did the same changes to SiteLocation, SiteIdentification and FormInformation.